ui/Dockerfile.dev

```diff
@@ -1,6 +1,8 @@
 # Note: This is the dockerfile for development purposes
 
-FROM node
+FROM node:14
+
+RUN apt-get update && apt-get -y install python
```

Comment:
Explicitly install Python because otherwise it can't be found when needed during npm install.

Reply:
Can you either leave a comment or specify explicitly in the code whether this is `python2` or `python3`?

It is confusing (and not your fault) that package managers/OSes use the same name even if they are installed concurrently!